torchci/scripts/check_alerts.py

```diff
@@ -31,47 +30,29 @@
 ISSUES_WITH_LABEL_QUERY = """
 query ($owner: String!, $name: String!, $labels: [String!]) {
   repository(owner: $owner, name: $name, followRenames: false) {
-    issues(last: 10, labels: $labels, states: [OPEN]) {
+    issues(last: 20, labels: $labels, orderBy: {field: UPDATED_AT, direction: ASC} ) {
```

Comment:
Let's keep an eye out for rate limit issue here as the state condition open is now removed.  This looks fine as the labels are still used.  I'm just being paranoid here :)